Examples/c++/Linked List/DLList.h

```diff
@@ -0,0 +1,401 @@
+#pragma once
+
+#include <algorithm>
+#include <utility>
+
+#ifdef DEBUG
```

Comment:
It's a great style to use assert, but I believe it's not standart for the course, so please add few comments which are going to help your collegues to undestand why you are doing this. This also could be done and at several other parts of the code below.

Reply:
I agree. Comments will be added explaining what the assert macro does.